packages/store/src/execution/dispatchOutsideZoneNgxsExecutionStrategy.ts

```diff
@@ -0,0 +1,52 @@
+import {
+  NgZone,
+  PLATFORM_ID,
+  Injectable,
+  ɵNoopNgZone as NoopNgZone,
+  Inject
+} from '@angular/core';
+import { isPlatformServer } from '@angular/common';
+
+import { NgxsExecutionStrategy } from '../symbols';
+import { ConfigValidator } from '../internal/config-validator';
+
+@Injectable()
+export class DispatchOutsideZoneNgxsExecutionStrategy implements NgxsExecutionStrategy {
```

Comment:
Maybe remove the `Ngxs` from the class name?

Reply:
I followed the convention of the `NgxsPlugin` interface to start with `Ngxs`.
So, because the interface is called `NgxsExecutionStrategy` the class would end with that by convention. ... ?

Reply:
See my comment below

Reply:
Rename `DispatchOutsideZoneNgxsExecutionStrategy` to `DispatchOutsideZone`

Reply:
Ok, then how would we name the `NoopNgxsExecutionStrategy`?

Reply:
`NoopNgZone`, `OutsideZoneNoop` ?

Reply:
The `NoopNgxsExecutionStrategy` has got nothing to do with Zones. The only thing that has something to do with zones is the `DispatchOutsideZoneNgxsExecutionStrategy`.

Reply:
ok